mautic/mautic
# [UX] Fixing page titles h-tag to improve accessibility

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
On the web, it’s common practice to use the H1 tag for the main title of a page for SEO and accessibility reasons. However, despite the hierarchy of titles (H1, H2, H3, etc.) may not have the same direct impact on SEO, as apps are not crawled by search engines in the same way web pages are, using a logical title structure is helpful for accessibility, as screen readers use this structure to assist users in navigation.

_Note! THIS INDIVIDUAL CHANGE WILL NOT CORRECT THE GROTESQUE ACCESSIBILITY PROBLEMS ON THE PLATFORM, AS OTHER CORRECTIONS ARE NECESSARY AND WILL ACT TOGETHER FOR A BETTER USE EXPERIENCE FOR USERS WITH DISABILITIES._



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Well, look at any title to see a subtle difference in its size (caused by the current CSS rules). If you are using a screen reader, you will notice that when you reach the page title it will read as level one (keep pressing tab unhurried until the reader reaches an item in the body and then use the arrow keys to try to reach the title).
3. /imagine How does a person with vision difficulties navigate the platform like this?
